Solutions/CyberArkEPM/Parsers/CyberArkEPM.txt

```diff
@@ -0,0 +1,92 @@
+// Usage Instructions: 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name as CyberArkEPM and specifying Legacy Category.
+// This function maps CyberArk EPM events to Azure Sentinel Information Model (ASIM) (https://docs.microsoft.com/en-us/azure/sentinel/normalization).
```

Comment:
Documentation links should not include locale: (https://docs.microsoft.com/en-us/azure/sentinel/normalization), please check and update.